src/packed-pack.ts

```diff
@@ -1,10 +1,10 @@
 import { promises as fsPromises, statSync } from 'fs';
 import { inflateSync } from './fast-inflate.js';
-import { BufferCursor, readEncodedOffset, readVarIntLE } from './utils/buffer.js';
```

Comment:
This is actually something that my ide does automagically :( 
Shall we settle on certain import order?

Reply:
I doubt it's necessary. However, I believe that this isn't a good practice to push a changes in an order of lines just because of author's IDE / plugin / etc does it automatically. This can result in changing the order of the lines back and forth on each push ;)